pipeline/Dockerfile

```diff
@@ -7,7 +7,7 @@ RUN apt-get update -y
 RUN apt-get install -y curl
 
 # Install ksonnet
-ARG KS_VERSION="0.11.0"
+ARG KS_VERSION="0.13.1"
```

Comment:
U sure this would cause any issue? there are integration test that covers deployment.

Reply:
Please refer to the comments at https://github.com/kubeflow/pipelines/issues/414#issuecomment-443607626 , I think the integration test is based on native Kubernetes but not other kubernetes distributions, like OpenShift, IBM Cloud Private etc.